gpflow/training/__init__.py

```diff
@@ -2,4 +2,8 @@
 
 from .scipy_optimizer import ScipyOptimizer
 from .hmc import HMC
+from .natgrad_optimizer import XiTransform
```

Comment:
Do we really need all of these imports to run the natgrad code?

Reply:
Not necessary. Will remove

Reply:
Actually not sure. Leaving them in

Reply:
hm, someone will decide to create their transforms, this is just a shortcut